docs/interfaces/postgres.rst

```diff
@@ -191,12 +191,14 @@ table available in CrateDB::
     | 1185 | _timestamptz                 |        0 |    1184 |     -1 | b       | A           |
     | 1186 | interval                     |     1187 |       0 |     16 | b       | T           |
     | 1187 | _interval                    |        0 |    1186 |     -1 | b       | A           |
+    | 1266 | timetz                       |     1270 |       0 |     12 | b       | D           |
```

Comment:
btw, do we need to document it in the data types documentation https://github.com/crate/crate/blame/master/docs/general/ddl/data-types.rst#L210?

Reply:
yes, on it